fastlane/lanes/release.rb

```diff
@@ -158,34 +288,52 @@
   #####################################################################################
   desc 'Updates store metadata and runs the release checks'
   lane :finalize_release do |options|
-    UI.user_error!('Please use `finalize_hotfix_release` lane for hotfixes') if android_current_branch_is_hotfix()
+    UI.user_error!('Please use `finalize_hotfix_release` lane for hotfixes') if android_current_branch_is_hotfix
+
+    ensure_git_status_clean
+    ensure_git_branch(branch: '^release/') # Match branch names that begin with `release/`
+
+    message = "Finalizing release: #{current_release_version}\n"
+    if options[:skip_confirm]
+      UI.message(message)
```

Comment:
Similar to a previous comment, it looks like we can always print out the message and then conditionally do the confirmation.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-Android/commit/6ab18efe9261a63b77766614c06c3bc661a697f6